<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tp2_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="planificationTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1111</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="clientTab">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QTableView" name="clientTableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>1101</width>
        <height>431</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="Chinese" country="China"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomCLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenomCLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="idLineEdit">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>10</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="selectCPushButton">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="French" country="France"/>
      </property>
      <property name="displayFormat">
       <string>yyyy/MM/dd</string>
      </property>
     </widget>
     <widget class="QLabel" name="nomLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nom:</string>
      </property>
     </widget>
     <widget class="QLabel" name="prenomLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prénom:</string>
      </property>
     </widget>
     <widget class="QLabel" name="idLabel">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Numéro d’identification: </string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>70</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy/MM/dd</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="dateCheckBox">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>80</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>80</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>à</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="personnelTab">
     <attribute name="title">
      <string>Personnel</string>
     </attribute>
     <widget class="QTreeView" name="personnelTreeView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>1101</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="reloadPushButton">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Reload...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Planification</string>
     </attribute>
     <widget class="QLabel" name="choisirDateLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choisir une date pour planifier:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="planifierDateEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy/MM/dd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="planifierPushButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>40</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Planifier</string>
      </property>
     </widget>
     <widget class="QTableView" name="planificationTableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>1111</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="enregistrerPushButton">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>40</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Enregistrer sous...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1109</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuAjouter">
     <property name="title">
      <string>Ajouter</string>
     </property>
     <addaction name="actionClient"/>
     <addaction name="actionAssureur_Vie"/>
     <addaction name="actionPersonnel"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuAjouter"/>
    <addaction name="actionQuitter"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionA_Propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPersonnel"/>
   <addaction name="separator"/>
   <addaction name="actionClient"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClient">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/myImages/client.jpg</normaloff>:/myImages/client.jpg</iconset>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionPersonnel">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/myImages/personnel.jpg</normaloff>:/myImages/personnel.jpg</iconset>
   </property>
   <property name="text">
    <string>Personnel</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionA_Propos">
   <property name="text">
    <string>A Propos</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAssureur_Vie">
   <property name="text">
    <string>Assureur Vie</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
